### ISSUE TITLE
$samplerate$ in .yml

### ISSUE THREAD

**Issue by mervinbeng** [HEnquist/camilladsp#128]
I'm close to having a functional equivalent of the old brutefirwrapper via camilladsp, but there is one issue:

In my config file, I tried using the $samplerate$ token so that I can support different sample rates:

```
  mbr_fir_l:
    type: Conv
    parameters:
      type: Wav 
      filename: /etc/drc/hall-phil3_48000.wav
      channel: 0
  mbr_fir_r:
    type: Conv
    parameters:
      type: Wav 
      filename: /etc/drc/hall-phil3_$samplerate$.wav
      channel: 1
```
The first reference was read correctly. The second gave this error:

ERRO Invalid filter 'mbr_fir_r'. Reason: No such file or directory (os error 2), module: camilladsp

Thanks for any pointers on this.

**Comment by HEnquist** [comment]
Thanks for opening this issue! This is a bug, the tokens aren't replaced when the file is a wav. I'll fix it today.

**Comment by HEnquist** [comment]
I have pushed a fix to the develop branch now. Can you please try it?

**Comment by mervinbeng** [comment]
Thanks Henrik.

I compiled the development version, but I got exactly the same result. This
is on aarch64 for a pi4.

FYI, I've created raw filters, and in raw format the $samplerate$ token
works.

Mervin

**Comment by HEnquist** [comment]
Hmm that is odd, it works when I try it.
Can you check line 932 of your src/config.rs to make sure it looks like this? https://github.com/HEnquist/camilladsp/blob/develop/src/config.rs#L932

If that is ok, then please start with -vv and post the output here.

**Comment by mervinbeng** [comment]
Oops... I had made a mistake on my github clone.

The develop branch works. Thanks very much.

In the process, I've uncovered another issue: in the develop branch there is a new problem when compiling for 32-bit operation:

```
$ cargo build --release --no-default-features --features alsa-backend --features 32bit 
   Compiling camilladsp v0.6.0 (/home/mervin/camilladsp-develop)
error[E0308]: mismatched types
  --> src/conversions.rs:62:39
   |
62 |         clipped += f64::write_samples(&nextframe, &mut cursor, &rawformat).unwrap();
   |                                       ^^^^^^^^^^ expected slice `[f64]`, found struct `Vec`
   |
   = note: expected reference `&[f64]`
              found reference `&Vec<f32>`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: could not compile `camilladsp`

To learn more, run the command again with --verbose.
```
The 64-bit build compiles without issue. The compilation is on arch linux aarch64.

**Comment by HEnquist** [comment]
Great!
And thanks for finding the 32-bit problem. I fixed the typo so it should work now.

**Comment by HEnquist** [comment]
Closed by version 0.6.0
